feature-libs/quote/components/details/comment/quote-details-comment.component.spec.ts

```diff
@@ -166,6 +191,7 @@ describe('QuoteDetailsCommentComponent', () => {
     quote.comments = [];
     quote.comments.push({});
     quote.comments.push({});
+
```

Comment:
agreed: please revert the addition of this empty line, as well as all of the following empty line additions.

Reply:
Done!

Reply:
done